pkg/agent/plugin/workloadattestor/unix/unix.go

```diff
@@ -241,6 +247,7 @@ func (p *Plugin) getUID(proc processInfo) (string, error) {
 func (p *Plugin) getUserName(uid string) (string, bool) {
 	u, err := p.hooks.lookupUserByID(uid)
 	if err != nil {
+		p.log.Warn("failed to lookup user name by uid", "uid", uid, "error", err)
```

Comment:
nit: log messages should start with uppercase

Reply:
Good catch!